crates/sncast/src/lib.rs

```diff
@@ -531,20 +533,20 @@ pub fn get_block_id(value: &str) -> Result<BlockId> {
 
 #[derive(Debug, CairoSerialize)]
 pub struct ErrorData {
-    pub data: String,
+    pub data: ByteArray,
 }
 
 impl ErrorData {
     #[must_use]
-    pub fn new(data: String) -> Self {
+    pub fn new(data: ByteArray) -> Self {
```

Comment:
I would delete this method since there's no need for it. It is merely a proxy for the constructor.